style.css

```diff
@@ -697,3 +697,39 @@ a:hover {
     margin-top: 2em;
     margin-bottom: 2em;
 }
+
+.informal_statement {
+    margin-top: 1em;
+}
+
+.informal_statement .explain {
+    color:var(--link-color);
+}
+
+.informal_statement .explain .tooltiptext {
+    visibility: hidden;
+    width: 120px;
+    background-color: black;
+    color: #fff;
+    text-align: center;
+    padding: 5px 0;
+    border-radius: 6px;   
+    
+    /* Position the tooltip text - see examples below! */
+    top: -5px;
+    left: 105%;
+
+}
```

Comment:
I assume the `explain` css class this refers to was used in a previous version of the HTML / in an in-development version?

Reply:
Yes, exactly.